docs/contributing.md

```diff
@@ -13,10 +13,9 @@ A trusted group of core developers are responsible for maintaining the PrairieLe
   - If you’d like to understand a PR better, ask questions rather than making assumptions.
 
 - **Responsibilities for reviewing**
+  - The primary reviewer is responsible for ensuring that the PR is in a satisfactory state for merging, and following up with the author. This includes testing the changes via the instructions in the PR description, and potentially indicating to the author that further testing is needed.
```

Comment:
I don't necessarily agree with this; IMO the burden of testing should always be on the author. The reviewer does of course have a role to play in e.g. pointing out potential gaps in testing.

Maybe we're agreeing but just with different phrasing? Let's discuss tomorrow.

Reply:
We agreed, just need to redo phrasing.

Reply:
Changed in c646e6d27eb56a0c6012330472f0b90717ddd50a. If you would like to make tweaks, you can make them directly.